.github/workflows/ci_pr.yml

```diff
@@ -12,12 +12,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
```

Comment:
This looks like good practice though, we should keep it :)

Reply:
it does come with the consequence of ~30s to download the whole history. But that is peanuts compared to the whole build time.